src/test/java/net/imagej/scripting/ImageJ2ScriptTest.java

```diff
@@ -212,4 +212,27 @@ public void testStackDirectoryImagesScript() throws InterruptedException, Execut
         Assert.assertNotNull(output);
     }
 
+ // I don't know how to test this script, since I need the IJ1 RoiManager object.
```

Comment:
Could we add ImageJ1 as a test dependency to solve that or am I missing a bigger point? (Sorry, for the stupid question, @hadim..)